libethereum/Executive.cpp

```diff
@@ -418,6 +444,14 @@ bool Executive::go(OnOpFunc const& _onOp)
 			m_excepted = toTransactionException(_e);
 			revert();
 		}
+		catch (AddressAlreadyUsed const& _e)
+		{
+			// EIP86
+			clog(StateSafeExceptions) << "Address already used. " << diagnostic_information(_e);
+			m_gas = 0;
+			m_excepted = toTransactionException(_e);
+			revert();
```

Comment:
well that how OutOfGas is handled right now... true that this is maybe not wanted (even for outOfGas)